### DISCUSSION TITLE
Te explico pendejo

### DISCUSSION THREAD

**Discussion by LidierRaccioppe** [LidierRaccioppe/JugandoConPullRequestByMaximo#1] [Q&A]
Sus preguntas putos

**Comment by LidierRaccioppe** [comment]
Escribe tus preguntas

**Comment by LidierRaccioppe** [comment]
Escribe tus preguntas

**Comment by TheMarcosasd** [comment]
Simple y sencillo no se que pasos tengo que hacer con el issue

**Comment by LidierRaccioppe** [comment]
Ve a la pestaña issue y crea uno nuevo

**Comment by LidierRaccioppe** [comment]
con nombre significativo

**Comment by TheMarcosasd** [comment]
Lo estoy leyendo con mucha sensualidad pero llevo dos putas lineas

**Comment by TheMarcosasd** [comment]
Hecho mainto

**Comment by LidierRaccioppe** [comment]
[imagen papa](https://pbs.twimg.com/media/Fs96G9BWABEwjQj.jpg)

**Comment by LidierRaccioppe** [comment]
haces el pull request (derecha de discuciones), haces tu rama y alteras la main, para luego mandarmelo
